Models/Connection/MySqlConnectionInfo.js

```diff
@@ -152,7 +163,9 @@ export default class MySqlConnectionInfo extends ConnectionInfo {
    * @throws {Error}
    */
   async executeProcedure(procedureName, parameters) {
-    const query = `CALL ${procedureName}(${Array.isArray(parameters) ? parameters.join(","): ""})`;
```

Comment:
Can we use `?` and avoid using string concatenation?

Reply:
fixed; thanks a lot